back/src/Controller/position.js

```diff
@@ -0,0 +1,88 @@
+// Constants
+let MIN_DISTANCE = 80;
+let MAX_PER_GROUP = 3;
+let NB_USERS = 10;
+
+// Utils
+let rand = function(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+// Test Data
+let users = [];
+for(let i = 1; i <= NB_USERS; i++) {
+    let user = {};
+    user.id = rand(0,99999);
+    user.X = rand(0, 60);
+    user.Y = rand(0, 60);
+    users.push(user);
+}
+
+// Compute distance between each user
+let computeDistance = function(users) {
+    let i = 0;
+    let distances = [];
+
+    users.forEach(function(user1, key1) {
+        users.forEach(function(user2, key2) {
+            if(key1 !== key2 && key1 < key2) {
```

Comment:
```suggestion
            if(key1 < key2) {
```

This should be enough (since if `key1 < key2`, then `key1 !== key2` )

Reply:
You got a point :D